### PR TITLE
DC-466: Shell Command: send netflow data

### PR DESCRIPTION
**Pull request by patrick-schweizer**
## Description
adds a shell command to manually send test flow data.

## Jira link(s)
- https://opennms.atlassian.net/browse/DC-466

### CODE REVIEW

File: minion/flows/parser/src/main/java/org/opennms/horizon/minion/flows/shell/SendFlowCmd.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        if (Files.exists(Paths.get(file))) {
+            dataToSend = Files.readAllBytes(Paths.get(file));
+        } else if(this.getClass().getResource("/flows/" + file) != null) {
+            dataToSend = toByteArray(this.getClass().getResourceAsStream("/flows/" + file));
+        } else {
+            System.out.printf("cannot read file %s. Please enter a valid file, e.g. 'netflow9_test_valid01.dat'.%n", file);
+            return null;
+        }
+
+        try ( DatagramSocket socket = new DatagramSocket()) {
```
Comment (freddy61025) [inline minion/flows/parser/src/main/java/org/opennms/horizon/minion/flows/shell/SendFlowCmd.java:75]: extra space

File: minion/flows/parser/src/main/java/org/opennms/horizon/minion/flows/shell/SendFlowCmd.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    @Override
+    public Object execute() throws Exception {
+
+        byte[] dataToSend;
+        if (Files.exists(Paths.get(file))) {
+            dataToSend = Files.readAllBytes(Paths.get(file));
+        } else if(this.getClass().getResource("/flows/" + file) != null) {
+            dataToSend = toByteArray(this.getClass().getResourceAsStream("/flows/" + file));
+        } else {
+            System.out.printf("cannot read file %s. Please enter a valid file, e.g. 'netflow9_test_valid01.dat'.%n", file);
```
Comment (freddy61025) [inline minion/flows/parser/src/main/java/org/opennms/horizon/minion/flows/shell/SendFlowCmd.java:71]: Capital letter

**Review by freddy61025 — APPROVED** [review]
LGTM and some really minor issues.

File: minion/flows/parser/pom.xml [hunk 3]
```diff
@@ -9,6 +9,22 @@
   <artifactId>flows-parser</artifactId>
   <name>OpenNms Horizon :: Minion :: Flows :: Parser</name>
   <packaging>bundle</packaging>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>5.1.2</version>
```
Comment (j-white) [inline minion/flows/parser/pom.xml:17]: we should be able to omit the version here, ideally we manage this in one place

**Review by FrancescoBuc86 — APPROVED** [review]

File: minion/flows/parser/src/main/java/org/opennms/horizon/minion/flows/shell/SendFlowCmd.java (hunk 2 above)
Comment (patrick-schweizer) [reply minion/flows/parser/src/main/java/org/opennms/horizon/minion/flows/shell/SendFlowCmd.java]: fixed

File: minion/flows/parser/pom.xml (hunk 3 above)
Comment (patrick-schweizer) [reply minion/flows/parser/pom.xml]: fixed (its actually defined in the grand parent pom)

**Pull request merged**